### ISSUE TITLE
720p resolution gives a black XBMC

### ISSUE THREAD

**Issue by GoogleCodeExporter** [davilla/atv-xbmc-launcher#15]
```
What steps will reproduce the problem?
1. ATV set to 720p resolution.
2. Start XBMC/Boxee or Coach surfer.
3. Screen is black.

What is the expected output? What do you see instead?
Would like to se XBMC GUI.
Black/Nothing.

What version of the product are you using? On what operating system?
ATV 2.3
atv-xbmc-launcher 0.8
atvusb-creator-1.0.b10 for Mac OS X (10.5.5)

Please provide any additional information below.
I have read in the forum that this is will be solved in version
atv-xbmc-launcher 0.9 
http://code.google.com/p/atvusb-creator/issues/detail?id=154#c24

Since I'm a bit anxious I wonder if this is a hard nut to crack or if we
will see a solution in the near future?  
```

Original issue reported on code.google.com by `jesper.a...@gmail.com` on 5 Dec 2008 at 1:50

**Comment by GoogleCodeExporter** [comment]
```
Probably a good idea to open an issue here, too. This is a known issue in 
XBMCLauncher 0.8; see this thread with 
informations on howto work around the problem: 
http://xbmc.org/forum/showpost.php?
p=247488&postcount=1
A completely new concept of Launching XBMC is in the works and will probably be 
ready for testing this 
weekend.


```

Original comment by `stephan.diederich@gmail.com` on 5 Dec 2008 at 1:54
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed in Launcher 2.1 which is available through 
XBMC/BoxeeLauncher->Update->Launcher 2.1 
http://xbmc.org/forum/showthread.php?t=42664


```

Original comment by `stephan.diederich@gmail.com` on 27 Dec 2008 at 12:19
- Changed state: **Fixed**
